apps/cowswap-frontend/src/modules/swap/services/swapFlow/index.ts

```diff
@@ -26,32 +25,24 @@ export async function swapFlow(
     return
   }
 
-  if (input.permitInfo && !input.hasEnoughAllowance) {
-    // If token is permittable and there's not enough allowance, get th permit hook
+  logTradeFlow('SWAP FLOW', 'STEP 2: send transaction')
+  tradeFlowAnalytics.trade(input.swapFlowAnalyticsContext)
```

Comment:
Just a nitpick, but probably we should send analytics event at least when permit is signed. If an user rejects the permit signing it doesn't make sense to track this event